lib/identity/hostdata/config_builder.rb

```diff
@@ -99,6 +130,15 @@ def build!(values)
         RedactedStruct.new(*@written_env.keys, keyword_init: true).
           new(**@written_env)
       end
+
+      def secrets_client
+        @secrets_client ||= Aws::SecretsManager::Client.new(
+          http_idle_timeout: 5,
```

Comment:
After testing on an instance, it looks like we need to supply the `region`
```suggestion
          region: Identity::Hostdata.aws_region,
          http_idle_timeout: 5,
```

Reply:
Thanks for catching!